lib/reporting/command_line_options.rb

```diff
@@ -8,6 +11,8 @@ def parse!(argv, out: STDOUT)
       verbose = false
       progress = true
       period = nil
+      slice = 3.hours
```

Comment:
maybe set set `slice` to nil here, and then later on we do like `slice ||= 3.hours` at the end to give it a default if it's unset?